### DISCUSSION TITLE
Using in Production?

### DISCUSSION THREAD

**Discussion by pietru2004** [bitbrain/pandora#177] [General]
I feel like it might be time to start trying use addon in projects that are not just concepts... When people go and use it they will start finding what might be wrong, I mean bugs...

**Comment by bitbrain** [comment]
I use it already in my RPG but until all bugs are resolved I'd advise against it, especially the data loss can be a huge issue if people don't use version control (many gamedevs don't)

**Reply by pietru2004** [reply]
Ok.
Also I think I figured way to prevent data corruption/loss, I mean use english as lang of editor.
